### DISCUSSION TITLE
Table Script

### DISCUSSION THREAD

**Discussion by Devmmcssi6** [supabase/supabase#2589] [Questions] [answered]
How to generate complete table script ? and how to take a backup of my table?

**Comment by silentworks** [comment, marked as answer]
You can get a backup of your database if you are on a pro plan from the **database/backups** section of the Supabase dashboard. Otherwise you can connect to the postgres database directly using any postgres client app and do a table/database backup from there, you can find the connection settings under **settings/database** in the Supabase dashboard.

**Reply by silentworks** [reply]
As stated in my previous message you can use any postgres client app and do this. The likes of TablePlus, DataGrip and others allow you to export your database.

**Comment by Devmmcssi6** [comment]
Hi Andrew,

We are using a free edition version. Can we get options like generate
scripts from it?
